shared/appengine.go

```diff
@@ -195,3 +200,18 @@ func getURL(host, path string, filter TestRunFilter) *url.URL {
 	detailsURL.RawQuery = filter.ToQuery().Encode()
 	return detailsURL
 }
+
+func (a appEngineAPIImpl) getGithubClientFromKey(key string) (*github.Client, error) {
```

Comment:
Nit; for consistency the parameter here is a `token`. In this context, a `key` is what we get back from the datastore.